src/ILAMB/Regions.py

```diff
@@ -257,6 +257,12 @@ def getMaskLatLon(self,label,var):
         mask : numpy.ndarray
             a boolean array appropriate for masking the input variable data
         """
+        try:
+            import rasterio
+            from rasterio import features
+        except:
```

Comment:
In the new ILAMB I am trying to follow better practices--learning a lot. In general it is not good to have a blank `except` here. It means that any exception would be caught, even if it isn't the one we intended. Let's change it to `except ImportError`. This way only that error will get caught, which is the behavior we except.